### PR TITLE
Fix/grid

### PR DESCRIPTION
**Pull request by antoniocapelo**
## Review template

The below is for reviewers of this PR to copy/paste into the review body and fulfill.

- Old grid break points: 0-599, 600-1095, 1095+
- New grid break points: 0-470, 471-552, 553-700, 701-899, 900-1196, 1197+

I have opened the preview link and:

- [ ] checked that this PR resolves the spec provided from trello / this description
- [ ] checked the effected pages at all breakpoints
- [ ] ensured that this PR does not introduce any obvious bugs

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -1,4 +1,4 @@
-
 CONTENTFUL_TOKEN=test
 CONTENTFUL_SPACE=space
 MEETUP_KEY=test
+GATSBY_ENVIRONMENT="development"
```
Comment (antoniocapelo) [inline .env.example]: Quick way to share environment variables into our gatsby build (https://www.gatsbyjs.org/docs/environment-variables/#client-side-javascript)

File: .storybook/addons.js [hunk 2]
```diff
@@ -1,4 +1,45 @@
 import '@storybook/addon-actions/register'
 import '@storybook/addon-links/register'
 import '@storybook/addon-a11y/register'
-import '@storybook/addon-viewport/register'
+import registerScissors from 'storybook-addon-scissors'
+
+const devices = [
```
Comment (antoniocapelo) [inline .storybook/addons.js]: I created this list of devices with widths inside the 'range' of each of our viewports (all different mobile, tablet sizes plus desktop)

File: netlify.toml [hunk 3]
```diff
@@ -1,2 +1,5 @@
 [build]
   Functions = "./functions"
+
+[context.deploy-preview.environment]
+  GATSBY_ENVIRONMENT = "preview"
```
Comment (antoniocapelo) [inline netlify.toml]: I'm using this `GATSBY_ENVIRONMENT` env var to check wether I should include some `dev` code, such as the GridDebugger. We may use this for other stuff in the future, such as enabling/disabling sourcemaps, etc

File: package.json [hunk 4]
```diff
@@ -67,7 +67,6 @@
     "@storybook/addon-actions": "4.1.11",
     "@storybook/addon-links": "4.1.11",
     "@storybook/addon-storyshots": "^4.1.11",
-    "@storybook/addon-viewport": "^4.1.11",
```
Comment (antoniocapelo) [inline package.json]: `viewport` doesn't allow for viewport creation

File: src/components/Common/Hr.js [hunk 5]
```diff
@@ -4,15 +4,16 @@ import is from 'styled-is'
 
 const Hr = styled.hr`
   height: ${remcalc(1)};
-  margin: ${remcalc(17)} 0 ${remcalc(18)} 0;
+  padding: ${remcalc(17)} 0 ${remcalc(18)} 0;
```
Comment (antoniocapelo) [inline src/components/Common/Hr.js]: with `margin` it was being hard to use this component near sections with a different color (it pushed the section down and a 'gap' between the 2 sections appeared. This solutions fixes it and allows for the same usage.

File: .storybook/addons.js (hunk 2 above)
Comment (SaraVieira) [reply .storybook/addons.js]: Nice!

File: netlify.toml (hunk 3 above)
Comment (SaraVieira) [reply netlify.toml]: 🎉

File: package.json (hunk 4 above)
Comment (SaraVieira) [reply package.json]: Funny

File: src/components/Common/GridDebugger.js [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+import { Col, Row, Grid } from '../grid'
+
+const Container = styled(Grid)`
+  position: fixed;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  opacity: 0.6;
+  z-index: ${props => props.theme.zIndexes.gridDebugger};
```
Comment (SaraVieira) [inline src/components/Common/GridDebugger.js]: fancy

File: src/components/Common/GridDebugger.js [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+
+  render() {
+    if (!this.state.showGrid) {
+      return null
+    }
+
+    return (
+      <Container>
+        <Row>
+          {Array.from(new Array(12)).map((el, idx) => (
```
Comment (SaraVieira) [inline src/components/Common/GridDebugger.js]: Nice

File: src/components/Common/Hr.js (hunk 5 above)
Comment (SaraVieira) [reply src/components/Common/Hr.js]: 👍

File: netlify.toml (hunk 3 above)
Comment (antoniocapelo) [reply netlify.toml]: 💃

File: src/components/Common/GridDebugger.js (hunk 6 above)
Comment (antoniocapelo) [reply src/components/Common/GridDebugger.js]: well yes indeed it is :)

File: src/components/Common/Hr.js [hunk 8]
```diff
@@ -4,15 +4,16 @@ import is from 'styled-is'
...
-  background-color: ${props => props.theme.colors.grey};
+  margin: 0;
+  border-top: 1px solid ${props => props.theme.colors.grey};
   ${is('short')`
     width: 60px;
   `};
   ${is('muted')`
     opacity: 0.25;
-    background-color: ${props => props.theme.colors.white};
+    border-color: ${props => props.theme.colors.white};
```
Comment (SaraVieira) [inline src/components/Common/Hr.js]: why? Just wondering why the change?

File: src/components/Training/Courses.js [hunk 9]
```diff
@@ -18,6 +18,8 @@ const Courses = ({ categories, toggleModal }) => {
             <SectionTitle noPadding>Course catalog</SectionTitle>
           </Padding>
         </Col>
+      </Row>
```
Comment (SaraVieira) [inline src/components/Training/Courses.js]: nice

File: src/components/Training/Modal.js [hunk 10]
```diff
@@ -24,7 +24,7 @@ const Wrapper = styled.div`
   opacity: 0;
   top: 0;
   left: 0;
-  z-index: 9999;
+  z-index: ${props => props.theme.zIndexes.modal};
```
Comment (SaraVieira) [inline src/components/Training/Modal.js]: When you start caring about z indexes you are a senior

File: src/utils/theme.js [hunk 11]
```diff
@@ -30,9 +31,21 @@ export const breakpoints = {
   desktop: 1197
 }
 
+export const elementSizes = {
+  tappableArea: 48
+}
+
+export const zIndexes = {
+  header: 999,
+  modal: 9999,
```
Comment (SaraVieira) [inline src/utils/theme.js]: maybe change modal to 1000 and grid debugger to 1001

File: src/components/Training/Modal.js (hunk 10 above)
Comment (antoniocapelo) [reply src/components/Training/Modal.js]: hahah, I've had my share with z-index problems so now I like to have the main ones in a central place.
Also, it's useful when you want to position things relative to each other like:

```
  z-index: ${props => props.theme.zIndexes.modal + 1};

```

File: src/utils/theme.js (hunk 11 above)
Comment (antoniocapelo) [reply src/utils/theme.js]: see, now I can change them all in the same file 😄

File: src/components/Training/Modal.js (hunk 10 above)
Comment (SaraVieira) [reply src/components/Training/Modal.js]: See? Senior

File: src/components/Common/Hr.js (hunk 8 above)
Comment (antoniocapelo) [reply src/components/Common/Hr.js]: because of what i said above? 

> with margin it was being hard to use this component near sections with a different color (it pushed the section down and a 'gap' between the 2 sections appeared. This solutions fixes it and allows for the same usage.

Comment (SaraVieira) [reply src/components/Common/Hr.js]: oh! Yeah! Changing to padding means you cant use the backgroud color anymore, right

File: src/components/Common/GridDebugger.js (hunk 7 above)
Comment (antoniocapelo) [reply src/components/Common/GridDebugger.js]: 😊

Comment (antoniocapelo) [reply src/components/Common/GridDebugger.js]: 😊
